src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -74,12 +65,19 @@ const PostNotificationChannelItem: (props: MessageChannelItemProps) => React.Rea
   }
 
   const postMaker = postNotifItem?.rawJson?.postMaker;
-  const isOwnPost = postNotifItem?.rawJson?.isOwnPost;
   const isOwnSignedPost = item?.rawJson?.isOwnSignedPost;
 
   const {determinePostType} = useChannelHook();
   const type = determinePostType(postNotifItem);
 
+  const getIsMe = () => {
+    if (item?.channelType === 'ANON_POST_NOTIFICATION')
+      return postNotifItem?.rawJson?.isOwnAnonymousPost;
+    if (item?.channelType === 'POST_NOTIFICATION') return postNotifItem?.rawJson?.isOwnSignedPost;
+
+    return false;
+  };
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a function `getIsMe` which checks the `channelType` and returns the corresponding `isOwnAnonymousPost` or `isOwnSignedPost` value. This seems like a good refactor as it encapsulates the logic for determining if the post belongs to the current user, improving readability.

However, please ensure that the `channelType` values 'ANON_POST_NOTIFICATION' and 'POST_NOTIFICATION' are always correctly set, as the function relies on these to return the correct result.



<!-- This is an auto-generated comment by OSS CodeRabbit -->